### DISCUSSION TITLE
Database autorefresh

### DISCUSSION THREAD

**Discussion by Sylvainjoon** [dbeaver/dbeaver#19607] [General]
Hi,

DBeaver allows to set autorefreshing period for a (particular) table.
https://dbeaver.com/docs/wiki/Data-Refresh/

Is it possible to set autorefresh for a whole shema / database?
With the community edition?

Thanks for your support !

**Comment by LonwoLonwo** [comment]
Nope, unfortunately, it is not possible.

**Reply by Sylvainjoon** [reply]
Thanks for your reply.

Is there another way, a trick to set up, an item to install to ensure we are all working on recent objects (tables) when we are working in a group on a database?
